<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vncTabWidgetClass</class>
 <widget class="QWidget" name="vncTabWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1968</width>
    <height>1162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vncTabWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>模型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>--model_dir=D:/workspace/PaddleDetection/output_inference/rtdetr_r50vd_6x_coco</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCv_one_ImgBtn">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveImgAndInsert">
       <property name="text">
        <string>保存图片和插入</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>保存路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SaveImg_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>D:\img\</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>显示器序号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_outputIndex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vncConnectbtn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveVMBtn">
       <property name="text">
        <string>移动虚拟机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveVMBtn2">
       <property name="text">
        <string>复位虚拟机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startVMBtn">
       <property name="text">
        <string>启动虚拟机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vncDisConnectbtn">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="keyMouseBtnOpen">
       <property name="text">
        <string>键盘鼠标连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="keyMouseBtnClose">
       <property name="text">
        <string>键盘鼠标断开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vncViewbtn">
       <property name="text">
        <string>观看</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="vncWidget" name="vnc_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1920</width>
           <height>1080</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1920</width>
           <height>1080</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>vncWidget</class>
   <extends>QWidget</extends>
   <header>vncwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
